lib/Runtime/Library/GlobalObject.cpp

```diff
@@ -1627,6 +1627,33 @@ namespace Js
 
         return function->GetScriptContext()->GetLibrary()->GetUndefined();
     }
+
+    //Write a copy of the current TTD log to a specified location
+    Var GlobalObject::EntryEmitTTDLog(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+        ARGUMENTS(args, callInfo);
+
+        TTDAssert(args.Info.Count == 2 && Js::JavascriptString::Is(args[1]), "Bad arguments!!!");
```

Comment:
>TTDAssert(args.Info.Count == 2 && Js::JavascriptString::Is(args[1]), "Bad arguments!!!"); [](start = 8, length = 89)

TTDAssert today is fatal error but if we change it to assert later all this type checking will be gone. Should we explicit have type checking as we do in other built-ins?